modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/resolver/ConnectionArgumentResolver.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.runtime.resolver;
+
+import org.mule.extension.runtime.OperationContext;
+
+public class ConnectionArgumentResolver implements ArgumentResolver<Object>
```

Comment:
Is this class needed? If yes, add javadoc and maybe could be renamed, as this looks as a NullArgumentResolver